attorch/module.py

```diff
@@ -23,9 +23,15 @@ def __len__(self):
         return len(self._modules)
 
     def __iter__(self):
-        return iter(self._modules.values())
+        return self._modules.keys()
 
     def keys(self):
         yield from self._modules.keys()
```

Comment:
I also meant to say that we should use `return` instead of `yield from` here and just pass through `keys` iterable from `self._modules.keys()`.